src/nav/full-stack-observability.yml

```diff
@@ -2163,36 +2157,32 @@ pages:
-              - title: Fluentd plugin
-                path: /docs/logs/enable-log-management-new-relic/enable-log-monitoring-new-relic/fluentd-plugin-log-forwarding
-              - title: Fluent Bit plugin
-                path: /docs/logs/enable-log-management-new-relic/enable-log-monitoring-new-relic/fluent-bit-plugin-log-forwarding
-              - title: Amazon Kinesis Data Firehose streaming
-                path: /docs/logs/enable-log-management-new-relic/enable-log-monitoring-new-relic/stream-logs-using-kinesis-data-firehose
-              - title: Kubernetes plugin
-                path: /docs/logs/enable-log-management-new-relic/enable-log-monitoring-new-relic/kubernetes-plugin-log-forwarding
-              - title: Logstash plugin
-                path: /docs/logs/enable-log-management-new-relic/enable-log-monitoring-new-relic/logstash-plugin-log-forwarding
-              - title: Vector plugin
-                path: /docs/logs/enable-log-management-new-relic/enable-log-monitoring-new-relic/vector-output-sink-log-forwarding
-              - title: Stream logs from Heroku
-                path: /docs/logs/enable-log-management-new-relic/enable-log-monitoring-new-relic/heroku-log-forwarding
-              - title: Azure logs ARM template
-                path: /docs/logs/enable-log-management-new-relic/enable-log-monitoring-new-relic/azure-log-forwarding
+          - title: Infrastructure agent logs
+            path: /docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent
+          - title: AWS Amazon Kinesis Data Firehose
+            path: /docs/logs/forward-logs/stream-logs-using-kinesis-data-firehose
+          - title: AWS FireLens plugin
+            path: /docs/logs/forward-logs/aws-firelens-plugin-log-forwarding
+          - title: AWS Lambda for Amazon S3
+            path: /docs/logs/forward-logs/aws-lambda-sending-logs-s3
+          - title: AWS Lambda for CloudWatch logs
+            path: /docs/logs/forward-logs/aws-lambda-sending-cloudwatch-logs
+          - title: Fluentd plugin
+            path: /docs/logs/forward-logs/fluentd-plugin-log-forwarding
+          - title: Fluent Bit plugin
+            path: /docs/logs/forward-logs/fluent-bit-plugin-log-forwarding
+          - title: Heroku syslog drains
+            path: /docs/logs/forward-logs/heroku-log-forwarding
+          - title: Kubernetes plugin
+            path: /docs/logs/forward-logs/kubernetes-plugin-log-forwarding
+          - title: Logstash plugin
+            path: /docs/logs/forward-logs/logstash-plugin-log-forwarding
+          - title: Microsoft Azure logs ARM template
+            path: /docs/logs/forward-logs/azure-log-forwarding
+          - title: Vector plugin
+            path: /docs/logs/forward-logs/vector-output-sink-log-forwarding
       - title: Logs in context
         path: /docs/logs/enable-log-management-new-relic/configure-logs-context/configure-logs-context-apm-agents
         pages:
```

Comment:
I think maybe this needs to be altered to the new page, it is set up as a redirect currently so will result in page not being accessible